node/src/container_chain_spawner.rs

```diff
@@ -58,11 +57,42 @@ use {
     tokio_util::sync::CancellationToken,
 };
 
-/// Struct with all the params needed to start a container chain node given the CLI arguments,
-/// and creating the ChainSpec from on-chain data from the orchestrator chain.
+/// Timeout to wait for the database to close before starting it again, used in `wait_for_paritydb_lock`.
+/// This is the max timeout, if the db is closed in 1 second then that function will only wait 1 second.
+const MAX_DB_RESTART_TIMEOUT: Duration = Duration::from_secs(60);
+
+/// Block diff threshold above which we decide it will be faster to delete the database and
+/// use warp sync, rather than using full sync to download a large number of blocks.
+/// This is only needed because warp sync does not support syncing from a state that is not
+/// genesis, it falls back to full sync in that case.
+/// 30_000 blocks = 50 hours at 6s/block.
+/// Assuming a syncing speed of 100 blocks per second, this will take 5 minutes to sync.
```

Comment:
is this assumption based of anything? I am glad to set this, we can tweak it through experimentation

Reply:
My main goal was to set it to so that it takes 5 minutes to sync, which is the session time in flashbox. More than that, and we get some downtime in flashbox, and less than that and we delete a potentially long chain only to download it again (because warp sync also downloads block history for now). But the 100 blocks per second is just a guess.

Also in the future we can make this value depend on the network (dancebox/flashbox) and on the size of the container chain if we find a way to estimate it.